app/templates/components/data-table-row-current.hbs

```diff
@@ -19,7 +19,7 @@
       class="cell-border-left no-compare-message"
       colspan={{if this.reliability "5" "2"}}
     >
-      Data are not available
+      Data are suppressed for selected area
```

Comment:
Are we sure this won't cause issues for other tables? Are there still cases where we do want it to say "Data are not available"?

Reply:
This only affects Decennial data sources. ACS data sources get a different message from [here](https://github.com/NYCPlanning/labs-factfinder/blob/develop/app/templates/components/data-table-row-previous.hbs#L142) or [here](https://github.com/NYCPlanning/labs-factfinder/blob/develop/app/templates/components/data-table-row-change.hbs#L211)